simple_tsify.js

```diff
@@ -0,0 +1,39 @@
+let through = require('through2')
+
+const isJson = (code) => {
+  try {
+    JSON.parse(code)
+  } catch (e) {
+    return false
+  }
+
+  return true
+}
+
+module.exports = function (b, opts) {
```

Comment:
Could you add some comments detailing what this is doing and why it's necessary?